src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int LENGTH_OF_BY = 4;
+    private static final int LENGTH_OF_AT = 4;
+    private static final int LENGTH_OF_EVENT = 6;
+    private static final int LENGTH_OF_DEADLINE = 9;
+    private static final int LENGTH_OF_TODO = 5;
+
     public static void main(String[] args) {
+        String raw_input;
+        Scanner scanner = new Scanner(System.in);
+        TasksList tasks = new TasksList();
+
+        printWelcomeBanner();
+
+        raw_input = scanner.nextLine();
+        while (true) {
+            if (raw_input.equals("bye")) {  // Exit programme
+                System.out.println("\tBye. Hope to see you again soon! \uD83D\uDC4B");  // Unicode is a waving hand
```

Comment:
I like the use of emojis in the print statements. It seems that you are printing emojis in multiple lines. Should they be declared as constants instead of being hardcoded?